test/test_utils.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
This cannot be run as is, only works on Raspberry PI, you can uncomment the appropriate sections, and we'll setup a docker to be able to run this on another PR